pkg/request/request.go

```diff
@@ -3,10 +3,13 @@ package request
 import "time"
 
 type Request struct {
-	Method    string            `json:"method"`
-	URI       string            `json:"uri"`
-	Body      []byte            `json:"body,omitempty"`
-	Header    map[string]string `json:"header"`
-	Status    int               `json:"status"` // status as reported by original server
-	Timestamp time.Time         `json:"ts"`     // time the request was created
+	Method     string            `json:"method"`
+	URI        string            `json:"uri"`
+	Body       []byte            `json:"body,omitempty"`
+	Header     map[string]string `json:"header"`
+	Status     int               `json:"status"` // status as reported by original server
+	Timestamp  time.Time         `json:"ts"`     // time the request was created
+	RemoteAddr string            `json:"remote_addr"`
+	UserAgent  string            `json:"agent"`
+	Referrer   string            `json:"referrer"`
```

Comment:
Can you make the field name and json tag `referer` - it's historically been spelled incorrectly but it's best to be consistent with the http header name.

Reply:
Sure, done! :)